common/src/main/java/net/william278/huskchat/config/Settings.java

```diff
@@ -278,6 +289,26 @@ private static Map<String, List<ChatFilter>> fetchChatFilters(YamlDocument confi
         return filters;
     }
 
+    /**
+     * Returns a map of Discord webhook URLs for each channel
+     * @param configFile The configuration file
+     * @return A map of webhook URLs for each channel
+     */
+    private static HashMap<String, URL> fetchWebhookUrls(@NotNull YamlDocument configFile) {
+        HashMap<String, URL> webhookUrls = new HashMap<>();
+        try {
+            for (String channelID : configFile.getSection("discord.channel_webhooks").getRoutesAsStrings(false)) {
```

Comment:
Check if this section actually exists (`configFile.contains()`) so that people updating to the next version of HuskChat don't get an NPE thrown here.

Reply:
Due to the way BoostedYaml's auto updater functions, this shouldn't be happen actually - but I guess better safe than sorry.

Reply:
Yeah, I thought so too but got an NPE when I tested this PR on my test server.